src/SprykerEco/Zed/AvalaraTax/Dependency/Plugin/CreateTransactionRequestExpanderPluginInterface.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\AvalaraTax\Dependency\Plugin;
+
+use Generated\Shared\Transfer\AvalaraCreateTransactionRequestTransfer;
+use Generated\Shared\Transfer\CalculableObjectTransfer;
+
+/**
+ * Use this plugin to extend `createTransaction` request before it's is send to the Avalara Tax API.
```

Comment:
* Use this plugin to extend `createTransaction` request before it's sent to the Avalara Tax API.